### PR TITLE
New gadgets (Struts2JasperReports - Atomikos - SpringJta)

### PR DESCRIPTION
**Pull request by sciccone**
Hi,

first big kudos for your outstanding research/work, I have used it in innumerable contexts and it was amazing.

I would like to add some gadget chains I have coded to the tool, if you agree:

- Struts2JasperReports. It's a gadget chain in the jasperreports plugin of struts2, which use a combination of Java deserialisation and OGNL evaluation. I have discovered it and coded the payload.

- Atomikos. It's a gadget chain ending with a JNDI lookup, already mentioned in some published research for JSON attacks (kudos to @pwntester). I tuned it a bit and coded the payload.

- SpringJta. It's a deserialised JNDI lookup, based on the work of @zerothoughts. I just coded the payload.

### CODE REVIEW

File: src/test/java/ysoserial/test/payloads/Struts2JasperReportsTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     *
+     */
+    public static final class StrutsJasperReportsDeserializer extends Deserializer {
+        
+		public StrutsJasperReportsDeserializer(byte[] bytes) {
+			super(bytes);
+		}
+    	
+        @Override
+        public Object call () throws Exception {
```
Comment (frohoff) [inline src/test/java/ysoserial/test/payloads/Struts2JasperReportsTest.java]: Is there a reason this context setup needs to be in a custom `Deserializer` class as opposed to just being in beginning of the payload `getObject()` implementation?

Comment (sciccone) [reply src/test/java/ysoserial/test/payloads/Struts2JasperReportsTest.java]: I have tried to put it within the `getObject` implementation, and if I run the `main` in the payload class works. But, when I run the complete tests, it seems the deserialisation appears in a different thread where the `ActionContext` has not been initialised. So, I created a custom `Deserializer` for doing that.

Comment (sciccone) [reply src/test/java/ysoserial/test/payloads/Struts2JasperReportsTest.java]: Hi @frohoff, do you have updates on merging this branch? Are there any problems?

Comment (frohoff) [reply src/test/java/ysoserial/test/payloads/Struts2JasperReportsTest.java]: No problems, just been super busy. Still hope to review more and merge. Thanks.

**Pull request merged**